bson/bsoncodec/registry.go

```diff
@@ -255,11 +255,10 @@ func (rb *RegistryBuilder) RegisterDefaultDecoder(kind reflect.Kind, dec ValueDe
 // mapping is decoding situations where an empty interface is used and a default type needs to be
 // created and decoded into.
 //
-// NOTE: It is unlikely that registering a type for BSON Embedded Document is actually desired. By
```

Comment:
Removed this warning as it's no longer needed. If the changes in this PR are accepted, ancestor information will always be used if available.